Unable to make on Ubuntu

#### Describe the bug
I installed libboost-dev zlib1g-dev libboost-python-dev and followed the installing procedure.
`cmake .. -DSTATIC_LINK_VW=ON` succeeded. But when I run `make -j` the following error showed:  

```
CMakeFiles/vw-unit-test.out.dir/main.cc.o: In function `main':
/usr/include/boost/test/unit_test.hpp:59: undefined reference to `boost::unit_test::unit_test_main(bool (*)(), int, char**)'
collect2: error: ld returned 1 exit status
test/unit_test/CMakeFiles/vw-unit-test.out.dir/build.make:164: recipe for target 'test/unit_test/vw-unit-test.out' failed
make[2]: *** [test/unit_test/vw-unit-test.out] Error 1
CMakeFiles/Makefile2:782: recipe for target 'test/unit_test/CMakeFiles/vw-unit-test.out.dir/all' failed
make[1]: *** [test/unit_test/CMakeFiles/vw-unit-test.out.dir/all] Error 2
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2
```

#### Environment
What version of VW did you use?
Most recent from git clone
What OS or language did you use?
Ubuntu 16.04.5 LTS

--- a/README.md
+++ b/README.md
@@ -140,7 +140,7 @@ git clone git://github.com/VowpalWabbit/vowpal_wabbit.git
 cd vowpal_wabbit
 mkdir build
 cd build
-cmake .. -DSTATIC_LINK_VW=ON
+cmake ..
 make -j
 make test       # (optional)
 make install

